src/components/Weather.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCitiesWeather } from '../redux/weatherSlice';
+import Nav from './Nav';
+import Cities from './Cities';
+import './styles/Weather.css';
+import arrow from './Assets/arrowRight.svg';
+
+const Weather = () => {
+  const cityData = useSelector((state) => state.weather.dataOfCities);
+  const isDataFetched = useSelector((state) => state.weather.isDataFetched);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (isDataFetched === false) {
+      dispatch(getCitiesWeather());
+    }
+  }, [dispatch, isDataFetched]);
+
+  return (
+    <div id="main-wrap">
+      <Nav />
+      <section className="cities">
+        {cityData.map((city) => (
+          <Link to={`city/${city.id}`} key={city.id}>
+            <div className="arrow-img"><img className="arrow" src={arrow} alt="arrow" /></div>
+            <Cities
+              city={city.city}
+              aqi={city.data.main.aqi}
+            />
+          </Link>
+        ))}
+      </section>
+    </div>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/zuhebahmed88091/AirVibe/assets/77069011/327663d3-50c2-4179-a4a3-43be5797d30d)

Please consider implementing a filter component so users can filter the cities' data. It could be, for example, a search bar that lets the user look for a city name! Please consider implementing this change 🤓 🔧